tests/ui/exact-harness/some_matching_harnesses/subset.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: --harness existing --harness check_blah --exact
+//! Check that we just ignore non-matching filters
```

Comment:
Unfortunately, our expected doesn't work for things that shouldn't show up. This test would still succeed if Kani had verified any set of harnesses that include the harness `existing`.

Maybe just use the same trick as you did with `ignored_harness` where the verification would fail if the harness was checked.

Reply:
Another possibility is to make the test run multiple harnesses, and include the `Complete - <N> successfully verified harnesses, 0 failures, <N> total.` with the correct number in the `expected` file.

@celinval perhaps Kani should always print the number of matched harnesses (even when it's just one)? This would make it easier to write expected files.

Reply:
I agree with that change, it would be useful for the extension as well.

Reply:
I like the idea of always printing the summary.

Reply:
+1 to printing the summary even for one harness.